packages/aws-cdk/bin/cdk.ts

```diff
@@ -185,7 +187,8 @@ async function initCommandLine() {
           exclusively: args.exclusively,
           templatePath: args.template,
           strict: args.strict,
-          contextLines: args.contextLines
+          contextLines: args.contextLines,
+          fail: args.fail || !configuration.context.get(cxapi.ENABLE_DIFF_NO_FAIL),
```

Comment:
Will this also work if this is defined in in cdk.context.json?

Reply:
Yes:
https://github.com/aws/aws-cdk/blob/6fa4713c1f07e41719701263f06f251e90819783/packages/aws-cdk/lib/settings.ts#L73-L76

Reply:
`DISABLE_DIFF_FAIL` instead of `ENABLE_DIFF_NO_FAIL`, I'm not very happy with the name here...